Vector/EmbeddingGenerator/embedder/text_embedder.py

```diff
@@ -0,0 +1,31 @@
+import openai
+import os
+import re
+import logging
+from tenacity import retry, wait_random_exponential, stop_after_attempt  
+
+class TextEmbedder():
+    openai.api_type = "azure"    
+    openai.api_key = os.getenv("AZURE_OPENAI_API_KEY")
+    openai.api_base = f"https://{os.getenv('AZURE_OPENAI_SERVICE_NAME')}.openai.azure.com/"
+    openai.api_version = os.getenv("AZURE_OPENAI_API_VERSION")
+    AZURE_OPENAI_EMBEDDING_DEPLOYMENT = os.getenv("AZURE_OPENAI_EMBEDDING_DEPLOYMENT")
+
+    def clean_text(self, text, text_limit=7000):
+        # Clean up text (e.g. line breaks, )    
+        text = re.sub(r'\s+', ' ', text).strip()
+        text = re.sub(r'[\n\r]+', ' ', text).strip()
```

Comment:
Combine those two?

Reply:
I'd like to leave this as-is for the time being...